database/migrations/2021_11_02_230344_create_roles_table.php

```diff
@@ -13,10 +13,10 @@ class CreatePasswordResetsTable extends Migration
      */
     public function up()
     {
-        Schema::create('password_resets', function (Blueprint $table) {
-            $table->string('email')->index();
-            $table->string('token');
-            $table->timestamp('created_at')->nullable();
+        Schema::create('roles', function (Blueprint $table) {
+            $table->id('roles_id');
```

Comment:
Lets make this simple as `id` so it will be `$table->id()` please update the other `id` as well ya @ahmadrizqialfa

Reply:
Ok untuk semua migration atau tabel ini doang?